src/table.c

```diff
@@ -2002,15 +2002,23 @@ void flecs_table_swap(
 
     ecs_type_info_t **type_info = table->type_info;
 
+    /* Find the maximum size of column elements
+     * and allocate a temporary buffer for swapping */
+    int32_t i, temp_buffer_size = sizeof(uint64_t), column_count = table->storage_count;
```

Comment:
Can you replace `sizeof` with `ECS_SIZEOF`? This should've thrown a warning, as it assigns an unsigned integer to a signed integer.
```c
temp_buffer_size = sizeof(uint64_t)
```